client/blocks/image-editor/image-editor-toolbar.jsx

```diff
@@ -179,21 +208,24 @@ class ImageEditorToolbar extends Component {
 			}
 		];
 
-		return buttons.map( button =>
-			button
+		return buttons.map( button => {
```

Comment:
We support Edge/IE 11 so if things work there, it's fair game. For the most part I'll usually just use lodash, so I don't need to think about it/worry about falsey items in the list.